selfdrive/car/honda/carcontroller.py

```diff
@@ -164,12 +164,12 @@ def update(self, CC, CS, now_nanos):
     can_sends = []
 
     # tester present - w/ no response (keeps radar disabled)
-    if self.CP.carFingerprint in (HONDA_BOSCH - HONDA_BOSCH_RADARLESS) and self.CP.openpilotLongitudinalControl:
+    if (self.CP.flags & HondaFlags.BOSCH) and (not self.CP.flags & HondaFlags.BOSCH_RADARLESS) and self.CP.openpilotLongitudinalControl:
```

Comment:
```suggestion
    if (self.CP.flags & (HondaFlags.BOSCH | HondaFlags.BOSCH_RADARLESS) == HondaFlags.BOSCH) and self.CP.openpilotLongitudinalControl:
```

works too, not sure if cleaner

Reply:
I think that's where the 'self.flags' is useful. I think having that everywhere is way too verbose